### PR TITLE
File Added : Uploaded Assignment 1,2,3,4,5,6

### PR DESCRIPTION
**Pull request by SnehalAhir**

### CODE REVIEW

File: Snehal/HTML/Assignment1to4/Assignment1/assignment12.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <title>Assignment 1</title>
+  </head>
+  <body>
+    <h1>HTML</h1>
+    <h3>Introduction</h3>
+    <p>HTML is a standard markup language for Web Pages.</p><hr>
+    <h3>HTML Editor</h3>
+    <h5>Notepad or TextEdit.</h5>
+    <p>Web pages can be created and modified by using professional HTML editors.<br>However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).</p>
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment1/assignment12.html]: remove extra line

File: Snehal/HTML/Assignment1to4/Assignment1/assignment12.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  </body>
+</html>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment1/assignment12.html]: add extra line at the end of the file

File: Snehal/HTML/Assignment1to4/Assignment1/assignment13.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  <head>
+    <title>Assignment 1</title>
+  </head>
+  <body>
+   <h2><b>ABOUT ME</b></h2><hr>
+   <p>My name is Snehal Ahir.I am 26yrs old from Jalgaon</p>
+   <h2>Qualification</h2><hr>
+   <p>I have completed BE, specialized in Computer Engineering in the year 2018, with 7.4 cgpa</p>
+  </body>
+</html>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment1/assignment13.html]: add extra line at the end of the file

File: Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html> [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Assignment-3_1</title>
+    </head>
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html>:6]: remove extra line

File: Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html> [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    <body>
+        <h2> Ambersoft</h2>
+        <p>
+            <ol > 
+               <li style="color: blueviolet;"> MI-MEAN
+                <ol style="list-style-type:lower-roman;" >  
+                    <li>MI-MEAN-6</li>
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html>:14]: remove this extra line space

File: Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html> [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+                    <ul>
+                        <li> Shweta </li> 
+                        <li>Ajay</li>
+                        <li>Trupti</li>
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html>:19]: remove this extra line space

File: Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html> [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+                    </ul>
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html>:21]: remove extra line space

File: Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html> [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+                               <li>Triveni</li>
+                               <li>Kalyani</li>
+                           </ul>
+                   </ol>
+                </li>   
+
+            </ol>
+        </p>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html>:60]: add extra line at the end of the file

File: Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html> [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+                          </ul>
+
+                        <li> SYSTEM-2</li>
+                           <ul>
+                               <li>Triveni</li>
+                               <li>Kalyani</li>
+                           </ul>
+                   </ol>
+                </li>   
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/<assignment_1. html>:56]: remove extra lines no need of extra lines here check everywhere

File: Snehal/HTML/Assignment1to4/Assignment3/assinment_2.html [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+            <img src= "/home/snehal/MI-11-MEAN/Snehal/HTML/index.jpeg" width='300' height='300'>
+            <p style="width: 900px; margin-left: 10px;"> we have heard that trees are our best friend but in practical life, we didn’t see anyone who treats trees as their friends. Although they are the most valuable life source on the earth. They benefit every life form in a direct or indirect way. And the earth is connected to them to maintain a natural balance. In this essay on save trees, we are discussing the reason why our friends need saving. 
+            </p>
+
+            <div style="text-align: right;">
+               <img src="/home/snehal/MI-11-MEAN/Snehal/HTML/index.jpeg" width='300' height='300'>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/assinment_2.html]: add extra line at the end of the file

File: Snehal/HTML/Assignment1to4/Assignment3/assinment_3.html [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+            </ul>
+
+            <br>
+
+            <ol>
+                <li>Darshu</li><li>Archu</li><li>Pratu</li>
+            </ol>
+        </dl>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/assinment_3.html]: add extra line at the end of the file

File: Snehal/HTML/Assignment1to4/Assignment3/assinment_4.html [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+        <h2 style="text-align: center;">Ambersoft</h2>
+
+        <p style="text-align: center;">  
+            <a href="file:///home/snehal/MI-11-MEAN/Snehal/HTML/assignment_1.html">Home</a> 
+            <a href="file:///home/snehal/MI-11-MEAN/Snehal/HTML/assignment_2.html"> MI-MEAN </a>| 
+            <a href="file:///home/snehal/MI-11-MEAN/Snehal/HTML/assignment_3.html"> Contact-Us</a>
+        </p>
+        <h3 style="text-align: center;"> Contact-us</h3>
+
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/assinment_4.html]: remove extra lines

File: Snehal/HTML/Assignment1to4/Assignment3/assinment_4.html [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+        <div class="container" style="display: flex;"> 
+            <img src= "/home/snehal/MI-11-MEAN/Snehal/HTML/index.jpeg" width='400' height='300'>
+
+            <p style="width: 900px; margin-left: 10px;"> 
+                we have heard that trees are our best friend but in practical life, we didn’t see anyone who treats trees as their friends. Although they are the most valuable life source on the earth. They benefit every life form in a direct or indirect way. And the earth is connected to them to maintain a natural balance. In this essay on save trees, we are discussing the reason why our friends need saving. 
+            </p>
+        </div>
+    </body>
+</html> |
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment3/assinment_4.html]: add extra line at the end of the file

File: Snehal/HTML/Assignment1to4/Assignment4/Second.html [hunk 14]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment2</title>
+    <style>
+        table, th, td{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Second.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Second.html [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+            border-left: 12px solid grey;
+            border-right: 12px solid grey;
+            border-top: 2px solid grey;
+            border-bottom: 2px solid grey;
+            padding: 15px;
+            border-collapse: collapse;
+        }
+    
+        .a{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Second.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Second.html [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+            border-top: 12px solid grey;
+        }
+
+        .b{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Second.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Second.html [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+            border-top: transparent;
+        }
+
+        .c{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Second.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Second.html [hunk 18]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Second.html]: check indentation

File: Snehal/HTML/Assignment1to4/Assignment4/Second.html [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+            border-top: transparent;
+        }
+
+        .c{
+            border-bottom: 12px solid grey;
+        }
+
+    </style>
+</head>
+<body style="margin: 50px 50px;">
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Second.html]: check indentation

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 20]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: check indentation

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        table{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+            border: 1px solid black;
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: remove extra line

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        table{
+            border: 1px solid black;
+            
+        }
+        th, td{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 24]
```diff
@@ -0,0 +1,87 @@
...
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid gainsboro;
+            border-right: 1px solid gainsboro;
+            padding: 5px;
+            border-width:2px;
+        }
+        .a{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 25]
```diff
@@ -0,0 +1,87 @@
...
+            border: 1px solid black;
+            
+        }
+        th, td{
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid gainsboro;
+            border-right: 1px solid gainsboro;
+            padding: 5px;
+            border-width:2px;
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before :

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 26]
```diff
@@ -0,0 +1,87 @@
...
+        }
+        .a{
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid gainsboro;
+            padding: 15px;
+        }
+        .b{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 27]
```diff
@@ -0,0 +1,87 @@
...
+            background-color:cornsilk;
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before :

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 28]
```diff
@@ -0,0 +1,87 @@
...
+        }
+        .c{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 29]
```diff
@@ -0,0 +1,87 @@
...
+            background-color:lightgray
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before :

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 30]
```diff
@@ -0,0 +1,87 @@
...
+        }
+        .d{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 31]
```diff
@@ -0,0 +1,87 @@
...
+            background-color:rgb(201, 245, 230);
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: space required before :

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 32]
```diff
@@ -0,0 +1,87 @@
...
+        }
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: remove extra line

File: Snehal/HTML/Assignment1to4/Assignment4/Third.html [hunk 33]
```diff
@@ -0,0 +1,87 @@
...
+        .c{
+            background-color:lightgray
+        }
+        .d{
+            background-color:rgb(201, 245, 230);
+        }
+        
+    </style>
+</head>
+<body style="margin: 30px 300px;">
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/Third.html]: check indentation

File: Snehal/HTML/Assignment1to4/Assignment4/first.html [hunk 34]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/first.html]: check indentation

File: Snehal/HTML/Assignment1to4/Assignment4/first.html [hunk 35]
```diff
@@ -0,0 +1,57 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment1</title>
+    <style>
+        caption{
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/first.html]: space required before curly bracket

File: Snehal/HTML/Assignment1to4/Assignment4/first.html [hunk 36]
```diff
@@ -0,0 +1,57 @@
...
+            font-size: x-large;
+            font-style: italic;
+            background-color:grey;
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/first.html]: space required before :

File: Snehal/HTML/Assignment1to4/Assignment4/first.html [hunk 37]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        table, th, td {
+            border: 1px solid black;
+            border-collapse: collapse;
+            background-color:grey;
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/first.html]: space required before :

File: Snehal/HTML/Assignment1to4/Assignment4/first.html [hunk 38]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        th, td {
+            padding: 15px;
+        }
+        
+
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/first.html]: remove extra lines

File: Snehal/HTML/Assignment1to4/Assignment4/first.html [hunk 39]
```diff
@@ -0,0 +1,57 @@
...
+    </style>
+</head>
+<body style="margin: 30px 300px;">
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment1to4/Assignment4/first.html]: check indenattion

File: Snehal/HTML/Assignment6/style.css [hunk 40]
```diff
@@ -0,0 +1,56 @@
...
+    margin-left: 95px;
+    margin-bottom: 20px;
+}
+#comment {
+    margin-left: 140px;
+}
+#submit-btn {
+    margin-top: 20px;
+    text-align: center;
+}
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment6/style.css]: add extra line at the end

File: Snehal/HTML/Assignment6/form2.html [hunk 41]
```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Question 2</title>
+        <link rel="stylesheet" href="./question_2.css" />
```
Comment (KomalShinde813) [inline Snehal/HTML/Assignment6/form2.html]: where is question_2.css file

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by jyoti-ing — APPROVED** [review]

**Review by SamikshaWani — APPROVED** [review]

File: Snehal/HTML/Assignment6/form2.html (hunk 41 above)
Comment (SnehalAhir) [reply Snehal/HTML/Assignment6/form2.html]: I have added it.

**Pull request closed without merging**